### PR TITLE
DOCSP-44635-global-cluster-limit

### PR DESCRIPTION
**Pull request by ajhuh-mdb**
## DESCRIPTION 
- Add to sharded cluster limitations that global clusters aren't supported as a source or destination

## STAGING 
https://deploy-preview-444--docs-cluster-to-cluster-sync.netlify.app/reference/limitations/#sharded-clusters:~:text=mongosync%20doesn%27t%20support%20sync%20to%20or%20from%20global%20clusters

## JIRA 
https://jira.mongodb.org/browse/DOCSP-44635

### CODE REVIEW

**Review by suspicious-mango — APPROVED** [review]
LGTM

File: source/reference/limitations.txt [hunk 1]
```diff
@@ -89,6 +89,8 @@ Sharded Clusters
   to a replica set.
 - ``mongosync`` doesn't support sync to a sharded cluster 
   topology with one or more arbiters.
+- ``mongosync`` doesn't support sync to or from :ref:`global clusters 
+  <global-clusters>`
```
Comment (jmd-mongo) [inline source/reference/limitations.txt:93]: [typo]

This needs a full stop at the end.

**Review by jmd-mongo — CHANGES_REQUESTED** [review]
Thanks for this addition, @ajhuh-mdb! I noted a typo.

Thanks,
Joe

File: source/reference/limitations.txt (hunk 1 above)
Comment (ajhuh-mdb) [reply source/reference/limitations.txt:93]: Ack! Thanks for catching that. Fixed.

**Comment by ajhuh-mdb** [comment]
Thanks for catching that typo @jmd-mongo. This should be ready for another pass.

**Review by jmd-mongo — APPROVED** [review]
LGTM!

Thanks,
Joe

**Comment by ajhuh-mdb** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|v1.8|[](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/446)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

**Pull request merged**